io/subsystem/src/main/java/org/wildfly/extension/io/IOSubsystemTransformers.java

```diff
@@ -44,15 +45,27 @@ public String getSubsystemName() {
     public void registerTransformers(SubsystemTransformerRegistration registration) {
         ChainedTransformationDescriptionBuilder chainedBuilder = TransformationDescriptionBuilder.Factory.createChainedSubystemInstance(registration.getCurrentSubsystemVersion());
 
-        // Current 3.0.0 to 2.0.0, aka EAP 7.0.0
-        buildTransformers_2_0(chainedBuilder.createBuilder(registration.getCurrentSubsystemVersion(), VERSION_2_0));
+        // Current 4.0.0 to 3.0.0, aka EAP 7.1.0
```

Comment:
I don't understand the mapping between versions.

Current is 3.0.0 (that you add in this PR), there is no 4.0.0, right?